html-aam/index.html

```diff
@@ -11911,7 +11914,10 @@ <h4 id="att-low">`low`</h4>
             <tr>
               <th><a href="https://developer.apple.com/reference/appkit/nsaccessibility">AX</a></th>
               <td>
-                <div class="general">Not mapped</div>
+                <div class="general">
+                  Expose "suboptial value" via `AXValueDescription` if &lt;value&gt; is greater than or equal to &lt;low&gt; and less than &lt;high&gt;. If &lt;value&gt; is less than &lt;low&gt;,
```

Comment:
[@rahimabdi's migrated comment](https://github.com/w3c/html-aam/pull/538/files#r1585788319):

Relevant WebKit code: https://github.com/WebKit/WebKit/blob/c330ea99229f6bb478dde74497ee652e44a3d7d5/Source/WebCore/html/HTMLMeterElement.cpp#L163

For low:

If value is less than or equal to low, then value is "optimal value"
if value is less than or equal to high (but greater than low), then value is "suboptimal value"
else, value is "critical value"
Also, looks like there is a typo for the word "suboptimal" here.